src/shared/VacationTable/VacationTable.tsx

```diff
@@ -149,32 +177,63 @@ const VacationTable: FC = () => {
               <TableCell sx={{ width: "150" }} align="center">
                 Type
               </TableCell>
-              <TableCell align="right">Start date</TableCell>
-              <TableCell align="right">End date</TableCell>
+              <TableCell align="center">Start date</TableCell>
+              <TableCell align="center">End date</TableCell>
               <TableCell sx={{ width: "500px" }} align="center">
                 Notes
               </TableCell>
               <TableCell align="center">Action</TableCell>
             </TableRow>
           </TableHead>
-          {rows.length < 1 ? null : (
+          {data.length < 1 ? null : (
```

Comment:
data.length === 0
In this case make something very simple to say "Table is empty"